### PR TITLE
Add UserManagement including db context, custom user model, razor account views and email service

### LINKED ISSUE #4: Account Management
Hinzufügen von App-Accounts und Verknüpfung dieser mit AzureAd-Logins. 
Hinzufügen einer Account-Verwaltung

### PR DESCRIPTION
**Pull request by Yorreck**

### CODE REVIEW

File: Data/ApplicationDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        public DbSet<Person> Persons { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            builder.Entity<Person>().ToTable("Person");
+
+            base.OnModelCreating(builder);
+            // Customize the ASP.NET Identity model and override the defaults if needed.
+            // For example, you can rename the ASP.NET Identity table names and more.
+            // Add your customizations after calling base.OnModelCreating(builder);
```
Comment (olohr) [inline Data/ApplicationDbContext.cs:31]: Leftover template comment

**Review by olohr — CHANGES_REQUESTED** [review]

File: Services/EmailSender.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        public EmailSender(IOptions<AuthMessageSenderOptions> optionsAccessor)
+        {
+            Options = optionsAccessor.Value;
+        }
+
+        public AuthMessageSenderOptions Options { get; } //set only via Secret Manager
+
+        public Task SendEmailAsync(string sentTo, string subject, string message)
+        {
+            // Hier den E-Mail-Dienst einfügen, um eine E-Mail-Nachricht zu senden.
```
Comment (olohr) [inline Services/EmailSender.cs]: We should keep all comments either german or english, I'd prefer english

**Review by olohr — CHANGES_REQUESTED** [review]

File: Data/ApplicationDbContext.cs (hunk 1 above)
Comment (Yorreck) [reply Data/ApplicationDbContext.cs:31]: Left comment for possible future enhancements and customization of identity model.

File: Services/EmailSender.cs (hunk 2 above)
Comment (Yorreck) [reply Services/EmailSender.cs]: comment now in english

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
...
+    {
+        "Authentication:AzureAd:ClientSecret": "<AzureAD ClientSecret",
+        "AuthMessageSenderOptions_Domain": "<Domain f�r SMTP Server>",
+        "AuthMessageSenderOptions_UserName": "<Nutzername f�r SMTP Server>",
+        "AuthMessageSenderOptions_SentFrom": "<Anzuzeigende Email-Adresse f�r SMTP Server>",
+        "AuthMessageSenderOptions_Password": "<Passwort f�r SMTP Server>",
+        "AuthMessageSenderOptions_SMTPClient": "<SMTP Client Adresse>"
+    }
+
+### Datenbank:
```
Comment (olohr) [inline README.md:28]: For linux the MSSQL database wont work, so I'd implement an option to use Mysql later, which is already implemented in the Schulungsportal

**Review by olohr — APPROVED** [review]

**Pull request merged**